ext/storage_sources/azure_store/README.md

```diff
@@ -0,0 +1,83 @@
+# WiredTiger's Azure Extension
+## 1. Introduction
+This extension allows WiredTiger storage source extensions to read from and write to objects stored
+in Azure Blob Storage using WiredTiger’s provided internal abstraction for storing data in an object
+storage service.
+
+## 2. Building and running
+This section describes how to build WiredTiger with the Azure extension enabled.
+
+### Requirements
+* CMake 3.13 or higher
+* G++ 8.4 or higher
+* clang version 6.0.0 or higher
+
+### Building
+
+There is currently only 1 way to build WiredTiger with Azure extension:
+1. Letting CMake manage the Azure SDK dependency as an external project, letting it download, link
```

Comment:
If we only have one way of it right now to know use a numbered list. I would say: 
```suggestion
There is currently only 1 way to build WiredTiger with Azure extension.
This way manages the Azure SDK dependency as an external project. This method will download the Azure SDK, and link to the WiredTiger's build system and build the extension.
```

Reply:
resolved in https://github.com/wiredtiger/wiredtiger/pull/8802/commits/0892029b180e2bbabfd909fd3d6c726e39b4f590